comms/rabbitmq.go

```diff
@@ -606,11 +633,17 @@ func (rmq rabbitMQComms) ListenForGroupTasks(groupName string, dereg chan bool)
 		rmq.config.AMQP.Port,
 	)
 
+	retries := 0
 	conn, err := amqp.Dial(queue_url)
-	if err != nil {
-		log.Error(err)
-		log.Error("Failed to connect to RabbitMQ")
-		os.Exit(1)
+	for err != nil {
+		if retries > connectRetry {
+			log.Error("Failed to connect to RabbitMQ")
+			return err
+		}
+
+		retries++
+		log.Warnf("Failure connecting to RabbitMQ - retry #%d", retries)
+		time.Sleep(1 * time.Second)
```

Comment:
Issue here too.

I don't think I did it in the retry loops I wrote, but you can initialize the retries variable in the for statement. Makes it a little bit cleaner imo.
``` go
for retries := 0; err != nil;  {
   ...
}